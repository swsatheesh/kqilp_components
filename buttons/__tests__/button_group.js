import React from 'react';
import renderer from 'react-test-renderer';
import ButtonGroup from '../button_group';

describe('components', () => {
  describe('<ButtonGroup />', () => {
    it('should render', () => {
      const buttonGroup = renderer.create(
        <ButtonGroup />
      ).toJSON();
      expect(buttonGroup).toMatchSnapshot();
    });
  });
});
