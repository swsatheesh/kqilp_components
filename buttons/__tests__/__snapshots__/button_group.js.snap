// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <ButtonGroup /> should render 1`] = `
<div
  className="btn-group"
/>
`;
